documentation/assemblies/metrics/assembly_metrics-prometheus-setup.adoc

```diff
@@ -3,26 +3,22 @@
 // deploying/deploying.adoc
 
 [id='assembly-metrics-setup-{context}']
-= Add Prometheus and Grafana
+= Using Prometheus with Strimzi
 
 You can use Prometheus to provide monitoring data for the example Grafana dashboards provided with Strimzi.
 
-In order to run the example Grafana dashboards, you must:
+To expose metrics in Prometheus format, you add configuration to a custom resource.
+You also need to deploy Prometheus and Prometheus Alertmanager.
```

Comment:
There are many different applications which can use Prometheus-compatible metrics. Can we maybe say something like:

_You will also need to make sure that the metrics are scraped by your monitoring stack. Our documentation and examples use Prometheus and Prometheus Alertmanager, but you can use also other compatible tools._

Reply:
Changed to:

> You will also need to make sure that the metrics are scraped by your monitoring stack. 
> Prometheus and Prometheus Alertmanager are used in the examples provided by Strimzi, but you can use also other compatible tools.